bali/db/models.py

```diff
@@ -38,8 +38,8 @@ def get_base_model(db):
     class BaseModel(db.Model):
         __abstract__ = True
 
-        created_time = Column(DateTime, default=datetime.utcnow)
-        updated_time = Column(DateTime, default=datetime.utcnow, onupdate=datetime.utcnow)
+        created_time = Column(AwareDateTime, default=timezone.now)
```

Comment:
`BaseModel` is used in many projects, we should changed it.
Maybe you can declare a `CustomBaseModel` in your project when customize